cmd/qat_plugin/README.md

```diff
@@ -76,7 +76,7 @@ For more details on the `-dpdk-driver` choice, see
 > `disable_denylist=1` parameter for the QAT device plugin to work correctly.
 
 For more details on the available options to the `-kernel-vf-drivers` option, see the list of
-vf drivers available in the [Linux Kernel](https://github.com/torvalds/linux/tree/master/drivers/crypto/qat).
+vf drivers available in the [Linux Kernel](https://github.com/torvalds/linux/tree/main/drivers/crypto/qat).
```

Comment:
@DougTW we also need to keep `master` here  (and one below) because Linux kernel has not moved